src/plugins/walletConnectToDapps/utils/clearAllWalletConnectToDappsSessions.ts

```diff
@@ -25,7 +27,4 @@ export const clearAllWalletConnectToDappsSessions = async () => {
       reason: getSdkError('USER_DISCONNECTED'),
     })
   }
-
-  // catch-all, clear local storage
-  clearWalletConnectLocalStorage()
```

Comment:
This was rugging the WC wallet state, and is only here as a catch-all to clear wallet-connect-to-dApp sessions, and shouldn't really be needed.